### DISCUSSION TITLE
Quick Links for relation fields/record

### DISCUSSION THREAD

**Discussion by bloor** [pocketbase/pocketbase#975] [Q&A] [answered]
A current issue I`m facing is , when I access a record I don;t have any direct-access to the relationed-records.
A link or "Ctrl-click" feature on it to open the linked record seems to be very helpful.

Now I have to copy the ID and manually search it in the relationed table.

![image](https://user-images.githubusercontent.com/1420653/200853136-19631a3e-631d-412d-b5f3-5c81c2cdcb45.png)

**Comment by ganigeorgiev** [comment, marked as answer]
I've created an issue so that we can track it and eventually add it to the roadmap - https://github.com/pocketbase/pocketbase/issues/976.

This may be added in the future and eventually combined with https://github.com/pocketbase/pocketbase/issues/881 and #100.
